### PR TITLE
fix: Download Tickets -> All Tickets

### LINKED ISSUE #5852: Order Success Page: Change "Download Tickets" to "All Tickets" and use download icon
On the order success page we have two buttons a) Download Ticket and b) "Download Tickets". This issue is about the button "Download Tickets". Please change the button to "All Tickets" and use a download icon for this specific button.

Screenshot from 2020-12-01 20-57-25

### PR DESCRIPTION
**Pull request by ksraj123**
Fixes #5852 

Screenshot 2020-12-02 at 2 05 37 AM

#### Changes proposed in this pull request:

- Download Tickets button changes to All Tickets
- Icon for button changes to download icon

Please share feedback and suggestions.

### CODE REVIEW

**Review by mariobehling — CHANGES_REQUESTED** [review]
There is a white space between the all tickets and invoice button now. Please keep the width of the button as it previously was. Thanks.

**Review by Raunaksingh100 — COMMENTED** [review]
Duplicate of #5860
I first say I am working on this issue so, why are  you spaming

**Review by cggvuhh — CHANGES_REQUESTED** [review]
there is lot of space and dupliacte of #5860 

 and #5860 already fixed the issue

**Comment by mariobehling** [comment]
Please be friendly to each other and collaborate. As mentioned elsewhere if two people collaborate and contribute code to an issue together it would also count for both in a coding contest like Codeheat for example.

**Comment by ksraj123** [comment]
@Raunaksingh100 The pr you mentioned was opened around 7 hours after this one. So how could this be a duplicate or spam. Also this tone of communication should not be encouraged in open source. As I already said in the issue I did not see your comment when I started working on this. If the mentors feel that this is already fixed then please feel free to close this.

**Comment by ksraj123** [comment]
@mariobehling Does this look good now? I have made the spacing like it was before.

Screenshot 2020-12-02 at 11 25 28 AM

**Review by mariobehling — APPROVED** [review]
Yes, looks good. Thank you!

Please keep the tone of the conversation positive and friendly. We are all here to learn as well.

File: app/templates/orders/view.hbs [hunk 1]
```diff
@@ -31,14 +31,14 @@
         @data={{this.model.order}} />
       <br/>
       {{#if (eq this.model.order.user.id  this.authManager.currentUser.id)}}
-        <div class="d-flex space-between">
-          <button class="ui labeled icon {{if this.isLoadingTickets 'loading'}} {{if this.device.isMobile 'fluid'}} button"
+        <div class="ui padded grid">
+          <button class="ui labeled icon left floated seven wide column {{if this.isLoadingTickets 'loading'}} {{if this.device.isMobile 'fluid'}} button"
```
Comment (iamareebjamal) [inline app/templates/orders/view.hbs:35]: Please use flexbox with different style. Your solution is not responsive

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
Please use flexbox as before

**Comment by ksraj123** [comment]
@iamareebjamal Unfortunately, I could not reproduce the responsiveness issues. Could you please share a screenshot for that.

Here is how it looked before on mobile devices -

original before changes

Here is how it looks now -

Screenshot 2020-12-02 at 6 56 47 PM

Also, I think under the hood, semantic UI grid uses flexbox only.

Screenshot 2020-12-02 at 7 01 06 PM


Please share feedback and suggestions. Thanks

**Comment by iamareebjamal** [comment]
> Could you please share a screenshot for that

You have done it above youself. See the difference carefully

**Comment by ksraj123** [comment]
Restored flexbox , now the space between the buttons not there on mobile if that was the resposivness issue we were talking about. Please have a look. Thanks

Screenshot 2020-12-02 at 7 25 32 PM
Screenshot 2020-12-02 at 7 25 57 PM

**Comment by iamareebjamal** [comment]
Not equal 
image

Remove justify stretch and set flex: 1 to both elements

**Comment by ksraj123** [comment]
Oh, Thanks. Buttons made equal now.

Screenshot 2020-12-02 at 7 51 02 PM

Should I squash my commits?

**Review by iamareebjamal — APPROVED** [review]

**Comment by iamareebjamal** [comment]
Thanks for the contribution

**Pull request merged**